spark3/src/main/java/org/apache/iceberg/spark/actions/BaseSnapshotTableSparkAction.java

```diff
@@ -109,7 +109,8 @@ public SnapshotTable tableProperty(String property, String value) {
 
   @Override
   public SnapshotTable.Result execute() {
-    JobGroupInfo info = newJobGroupInfo("SNAPSHOT-TABLE", "SNAPSHOT-TABLE");
+    JobGroupInfo info = newJobGroupInfo("SNAPSHOT-TABLE",
+        String.format("Snapshotting table %s(location=%s)", sourceTableIdent().toString(), destTableLocation));
```

Comment:
I think it is important to show the dest table identifier. Also, the location can be null if not provided. Since we are not showing table props and the location can be long, I'll be fine just saying this:

```
Snapshotting table %s as %s
```